src/dv360.ts

```diff
@@ -190,4 +191,84 @@ export class DV360 extends ApiClient {
     }
     return campaigns;
   }
+
+  protected createSdfDownloadOperation(advertiserId: string) {
+    const downloadOperation = this.fetchEntity(
+      this.getUrl('sdfdownloadtasks'),
+      {
+        method: 'post'
+      },
+      {
+        version: 'SDF_VERSION_5_5',
+        advertiserId: advertiserId,
+        parentEntityFilter: {
+          fileType: [
+            'FILE_TYPE_CAMPAIGN',
+            'FILE_TYPE_INSERTION_ORDER',
+            'FILE_TYPE_LINE_ITEM',
+            'FILE_TYPE_AD_GROUP',
+            'FILE_TYPE_AD'
+          ],
+          filterType: 'FILTER_TYPE_NONE',
+          filterIds: []
+        }
+      }
+    );
+    return downloadOperation;
+  }
+
+  protected waitForDownloadOperationResource(downloadOperationName: string) {
+    const operationResourceUrl = this.getUrl(downloadOperationName);
+    const initialDelay = 2000;
```

Comment:
Could initialDely, maxRetries and delayMultiplier be optional parameters with default values? I.e.
`waitForDownloadOperationResource(downloadOperationName: string, initialDelay: 2000, maxRetries=10, delayMultiplier=2)`

Reply:
Like it